dev/_testcurl.html

```diff
@@ -16,11 +16,12 @@
 TITLE(testcurl.1 the man page)
 <div class="relatedbox">
 <b>Related:</b>
+<br><a href="tests-overview.html">Tests Overview</a>
 <br><a href="runtests.html">runtests.1</a>
 <br><a href="/docs/manual.html">Manual</a>
 </div>
 
-#include "testcurl.gen"
+#include "tests-overview.gen"
```

Comment:
Really? Shouidn't this still show the testcurl man page? Also, this file name has no dash does it?